Ontology/Willow/Asset/Asset.json

```diff
@@ -318,24 +279,6 @@
       "schema": "string",
       "writable": true
     },
-    {
-      "@type": "Property",
```

Comment:
We also need to keep serialNumber and software as base properties.

Reply:
These properties have been restored.